### PR TITLE
Chore: update the CHANGELOG and README and bump to 2.0.0

### PR DESCRIPTION
**Pull request by spencern**
Updates the CHANGELOG and the README in preparation for the release of 2.0.0

Also bumps the `package.json` to 2.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,57 @@
+# v2.0.0
+
+Reaction v2.0.0—an API-first, real-time commerce engine built using Node.js, React, and GraphQL. It’s the second major release of our open source commerce software.
+
+This release is coordinated with [Reaction Platform](https://github.com/reactioncommerce/reaction-platform) and is designed to work with the [Reaction](https://github.com/reactioncommerce/reaction) and `[reaction-hydra](https://github.com/reactioncommerce/reaction-hydra)`.
+
+Reaction v2.0.0 is built as a truly headless commerce platform that decouples the Reaction backend services from the frontend. We’ve decoupled the storefront application from the API. Reaction platform now consists of this [Example Storefront](https://github.com/reactioncommerce/example-storefront), which connects with the Reaction application via GraphQL to provide a customer-facing storefront. The API is provided by the [Reaction](https://github.com/reactioncommerce/reaction) project, which is now primarily our GraphQL API. The legacy integrated Meteor storefront is no longer part of the Reaction project.
```
Comment (willopez) [inline CHANGELOG.md]: how about changing `connects` => `integrates` the storefront is an integration with the API

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+
+
+### Share your feedback
+We want to hear from you! Here are some good ways to get in touch.
+- Want to request a new feature for Reaction? There’s now a Reaction repo just for [new feature requests](https://github.com/reactioncommerce/reaction-feature-requests).
+- Reaction engineers and community engineers and developers are always collaborating in our [Gitter chat channel](https://gitter.im/reactioncommerce/reaction)
+- Ask Us Anything! Watch this space for details about an upcoming Community Q&A session with the Reaction team.
+
 # v2.0.0-rc.12
 This is our fourth release candidate for this project. While this project is technically still `pre-release` until we've released the final 2.0.0 version, it's the most stable version of Reaction Commerce, and we recommend starting new projects with it at this point. See https://github.com/reactioncommerce/example-storefront/issues/487#issuecomment-507468894 for more detail on this.
```
Comment (willopez) [inline CHANGELOG.md]: I believe this should be, `this is our twelfth` release candidate.

Comment (willopez) [reply CHANGELOG.md]: actually this whole RC section should be removed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,57 @@
+# v2.0.0
+
+Reaction v2.0.0—an API-first, real-time commerce engine built using Node.js, React, and GraphQL. It’s the second major release of our open source commerce software.
+
+This release is coordinated with [Reaction Platform](https://github.com/reactioncommerce/reaction-platform) and is designed to work with the [Reaction](https://github.com/reactioncommerce/reaction) and `[reaction-hydra](https://github.com/reactioncommerce/reaction-hydra)`.
```
Comment (willopez) [inline CHANGELOG.md]: hydra link is not rendering correctly. See preview here: https://github.com/reactioncommerce/example-storefront/blob/4080d68f545050faa3633f613a24a335fdeade0b/CHANGELOG.md

File: CHANGELOG.md (hunk 2 above)
Comment (spencern) [reply CHANGELOG.md]: We don't want to remove old bits from the changelog, but you're correct that it should be fixed. That must have slipped into our last release.

Comment (spencern) [reply CHANGELOG.md]: Looking into this, this was the 4th candidate for this project, because we didn't start tagging releases in the storefront until rc9

**Comment by spencern** [comment]
I believe these snyk tests are hanging because we renamed the project. I'll fix on the `snyk` side.

**Pull request merged**